Added sentiment udf (#385)
exasol/transformers-extension: doc/user_guide/user_guide.md

The extension provides two types of UDFs:

> * Utility UDFs: Managing of Transformers models in the Exasol BucketFS.

Reviewer: more with the verb for similar vibe
```suggestion
* Utility UDFs: Manage Transformers models in the Exasol BucketFS.
```

For prerequisites and the setup-guide, please visit the [Getting started](setup.md) file.

### Utility UDFs

This Exasol Extension provides UDFs for interacting with Hugging Face's Transformers API to use pre-trained models on an Exasol cluster.

These UDFs deal with the installation and deletion of pretrained Transformers models in the Exasol BucketFS.

We provide the following UDFs:

| UDF Name                     | Use                                                            |
|------------------------------|----------------------------------------------------------------|
| TE_MODEL_DOWNLOADER_UDF      | Download a specific model from Hugging Face.                   |
| TE_LIST_MODELS_UDF           | List all models available in the BucketFS.                     |
| INSTALL_AI_DEFAULT_MODEL_UDF | Install all default models used in the Transformers Extension. |
| TE_DELETE_MODEL_UDF          | Delete a specific model from the BucketFS.                     |

You can find further information on these UDFs [here](manage_models.md).

### Prediction UDFs

> These UDFs call a model stored in the BucketFS, and use it to make predictions on the given input data.

Reviewer: ```suggestion
These UDFs call a model stored in the BucketFS and use it to make predictions on the given input data.
```

However, if you want to configure a task to your specific needs, 
UDFs with the suffix "Extended" in the name allow you to specify all available 
parameters for each input row. 

For example, you may want to select a specific model to be used. 

These are the available UDFs:

| UDF Name                    | task_type                | Use                                                                  |
|-----------------------------|--------------------------|----------------------------------------------------------------------|
| AI Custom Classify Extended | text-classification      | Classifies the given text into classes known to the model.           |
| AI Entailment Extended      | text-classification      | Takes two input texts and compares them.                             |
| AI Answer Extended          | question-answering       | Extracts answer(s) from a given question text.                       |
| AI Fill Mask Extended       | fill-mask                | Replace ```<mask>``` tokens in the input with predicted text.        |
| AI Complete Extended        | text-generation          | Predict the continuation of the given text.                          |

> | AI Extract Extended         | token-classification     | Find and lable tokens in a given text.                               |

Reviewer: ```suggestion
| AI Extract Extended         | token-classification     | Find and label tokens in a given text.                               |
```
